spring-test/src/test/java/org/springframework/test/web/servlet/samples/client/standalone/resultmatches/XpathAssertionTests.java

```diff
@@ -224,11 +224,14 @@ public class BlogFeedController {
 		@RequestMapping(value = "/blog.atom", method = {GET, HEAD})
 		@ResponseBody
 		public String listPublishedPosts() {
-			return "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n"
-					+ "<feed xmlns=\"http://www.w3.org/2005/Atom\">\r\n"
-					+ "  <title>Test Feed</title>\r\n"
-					+ "  <icon>https://www.example.com/favicon.ico</icon>\r\n"
-					+ "</feed>\r\n\r\n";
+			return """
+					<?xml version="1.0" encoding="UTF-8"?>\r
+					<feed xmlns="http://www.w3.org/2005/Atom">\r
+						<title>Test Feed</title>\r
+						<icon>https://www.example.com/favicon.ico</icon>\r
+					</feed>\r
+					\r
+					""";
```

Comment:
Instead of explicitly adding `\r` to the end of each line, how about `""".replaceAll("\n", "\r\n");`?